Fix put rollup job api documentation (#3637)
opensearch-project/documentation-website: _im-plugin/index-rollups/rollup-api.md

> `error_notification` | Set up a Mustache message template sent for error notifications. For example, if an index rollup job fails, the system sends a message to a Slack channel. | Object | No

Reviewer: ```suggestion
`error_notification` | Set up a Mustache message template for error notifications. For example, if an index rollup job fails, the system sends a message to a Slack channel. | Object | No
```

> `page_size` | Specify the number of buckets to paginate through at a time while rolling up. | Number | Yes

Reviewer: ```suggestion
`page_size` | Specify the number of buckets to paginate at a time during rollup. | Number | Yes
```
